### DISCUSSION TITLE
Suggestion to base file

### DISCUSSION THREAD

**Discussion by steam3d** [WeblateOrg/weblate#6522] [Q&A]
Hi. Can i set only "add suggestion" to base file without allow editing this file?
![image](https://user-images.githubusercontent.com/30986874/132253256-00d5a0cf-7e79-4aa2-adfb-32c910f2b266.png)

**Comment by nijel** [comment]
See https://docs.weblate.org/en/latest/admin/projects.html#project-source-review
